samtranslator/model/sam_resources.py

```diff
@@ -1120,7 +1120,8 @@ class SamStateMachine(SamResourceMacro):
         "Name": PropertyType(False, is_str()),
         "Type": PropertyType(False, is_str()),
         "Tags": PropertyType(False, is_type(dict)),
-        "Policies": PropertyType(False, one_of(is_str(), is_type(dict), list_of(one_of(is_str(), is_type(dict))))),
+        "Policies": PropertyType(False, one_of(is_str(), list_of(one_of(is_str(), is_type(dict), is_type(dict))))),
+        "Tracing": PropertyType(False, is_type(dict)),
```

Comment:
Looks like original property is called `TracingConfiguration`, but in SAM it is referred as `Tracing`, should we keep same name since this is a pass-through property?

Reply:
@mndeveci We decided to name this property `Tracing` following our existing convention of naming configuration properties to the shorter name. The `Logging` property in SAM State Machine resource passes the value to the `LoggingConfiguration` property in CloudFormation State Machine resource.